engine/core/IgeEntity.js

```diff
@@ -3843,11 +3843,11 @@ var IgeEntity = IgeObject.extend({
 		}
 
 		if (isUnitExists) {
-			// remove all passive attributes applyed to this unit
+			// remove all passive attributes applied to this unit
 			unit._stats.itemIds.forEach(function (itemId) {
 				if (itemId) {
 					var item = ige.$(itemId);
-					if (item._stats.bonus && item._stats.bonus.passive) {
+					if (item._stats && item._stats.bonus && item._stats.bonus.passive) {
```

Comment:
looking at the issue, seems like `item` was undefined, and not `_stats`

![image](https://user-images.githubusercontent.com/7340300/180222978-5c4a42a9-6ad4-4242-98df-151c68a3be1e.png)

Reply:
@pavle-goloskokovic My bad, updated the PR.
Also added 'configure' method under rollbar for local support, we will be using this method to send game and server data to rollbar with logs.